highest-number/solutions/highest-number.ts

```diff
@@ -1,3 +1,3 @@
 export function getHighestNumber(numbers: number[]): number {
-  return numbers.slice().sort()[numbers.length - 1]
+  return numbers.reduce((acc: number, value: number) => (value > acc ? value : acc))
```

Comment:
I don't think there is need to specify the. type, as it can be inferred from the parameter type. Let's try removing it.